src/pacientes/pacienteController.ts

```diff
@@ -47,7 +50,16 @@ export const pacienteGet = async (req: Request, res: Response): Promise<void> =>
   const { id } = req.params
   const paciente = await AppDataSource.manager.findOne(Paciente, {
     where: { id },
-    relations: ['endereco']
+    select: {
+      id: true,
+      nome: true,
+      email: true,
+      telefone: true,
+      planoSaude: true
+    },
```

Comment:
Essa lógica aqui existe pra que não tenha a senha sendo retornada. Funciona mas, do jeito que ta feito é muito frágil. em uma nova requisição pode ser que alguém esqueça de fazer essa validação e a senha seja retornada. provavelmente o TypeORm tem alguma anotation pra deixar isso registrado na Entidade e a senha não ser retornada

Reply:
achei isso aqui, vale testar https://www.kindacode.com/article/typeorm-2-ways-to-exclude-a-column-from-being-selected/